awx_collection/meta/runtime.yml

```diff
@@ -1,46 +1,123 @@
 ---
 action_groups:
-  tower:
-    - tower_credential_input_source
-    - tower_credential
-    - tower_credential_type
-    - tower_group
-    - tower_host
-    - tower_inventory
-    - tower_inventory_source
-    - tower_job_cancel
-    - tower_job_launch
-    - tower_job_list
-    - tower_job_template
-    - tower_job_wait
-    - tower_label
-    - tower_license
-    - tower_notification
-    - tower_organization
-    - tower_project
-    - tower_role
-    - tower_schedule
-    - tower_settings
-    - tower_team
-    - tower_token
-    - tower_user
-    - tower_workflow_job_template_node
-    - tower_workflow_job_template
-    - tower_workflow_launch
+  controller:
```

Comment:
The use case for this looks like:

```yaml
---
- hosts: localhost
  gather_facts: false
  connection: local
  collections:
    - awx.awx
  module_defaults:
    awx.awx.controller:
      tower_host: https://foo.invalid
      organization: Default
  tasks:
    - name: ...
```

If you're going to be vague but count on FQCN use, you might as well name it "default" or something. If you're trying to use a name such that people recognize the product, I think "controller" is _too_ vague and borderline self-contradictory sitting next to `awx.awx`. Maybe "ansible_controller". Even "tower" was always too vague of an abbreviation when put into its proper use context IMO.

Reply:
I believe the idea is to rename the collections to `ansible.controller`, I need to update the code to reflect that

Reply:
An `ansible.*` name is for the downstream. This `awx.awx` name is for the upstream.

We can't have split names for things _inside_ the collection, and we really shouldn't have had 2 different names for the upstream / downstream collection in the first place. The framework for collections works very bad. We template code in some places.

If you use the recent ansible-core and run `ansible-test` without our ignore list, you will find it littered with errors because all of the warnings and deprecations are version-aware, but we are supporting two different versioning systems. This is broken, and it was never fixed. We have skated by because we largely failed to test with the recent Ansible versions.

While a name might sound superficial, it's actually a monster of a problem to go from the intended naming scheme to something that works and actually confirms to present standards for collections.